.github/workflows/build.yml

```diff
@@ -276,6 +276,68 @@ jobs:
         if: ${{ steps.validation.outcome == 'failure' }}
         run: exit 1
 
+  verify-packages:
+    strategy:
+      matrix:
+        node: [ 18 ]
+      fail-fast: true
+      max-parallel: 1
+    name: Verify packages are installable on node ${{ matrix.node }}
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Setup repo
+        uses: actions/checkout@v3
+
+      - name: Build all packages
+        run: |
+          pushd packages/core
+            npm install
+            npm run build
+            npm pack
+          popd
+          pushd packages/common-integration-tests
+            npm install
+            npm run build
+            npm pack
+          popd
+          pushd packages/client-sdk-nodejs
+            npm install
+            npm run build
+            npm pack
+          popd
+          pushd packages/client-sdk-web
+            npm install
+            npm run build
+            npm pack
+          popd
+
+      - name: Locally install in nodejs example and verify it builds
+        run: |
+          pushd examples/nodejs/cache
+            # This is a bit of a hack, but ensures that the package-lock does not get corrupted
+            # due to installing these locally packaged files. All we care about is that the examples
+            # use the locally packaged sdks, and that they can build correctly with no errors
+            rm -rf node_modules package-lock.json
+            npm i ../../../packages/core/gomomento-sdk-core-0.0.1.tgz
+            rm -rf node_modules package-lock.json
+            npm i ../../../packages/client-sdk-nodejs/gomomento-sdk-0.0.1.tgz
+            rm -rf node_modules package-lock.json
+            npm i
+            npm run build
```

Comment:
@bruuuuuuuce maybe think about in follow up pr or issue be good to maybe try running example or run some test also make sure can actually start up? At least making sure builds for now is better then nothing here.